src/js/components/ExtraInfo/extraInfo.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ExtraInfo from './index';
+
+const props = {
```

Comment:
It is nice to fill props with data in `beforeEach` hook, so that If one test modifies them - it won't affect other tests